SU2_CFD/src/solvers/CDiscAdjFEASolver.cpp

```diff
@@ -418,24 +367,18 @@ void CDiscAdjFEASolver::SetSensitivity(CGeometry *geometry, CConfig *config, CSo
 
   /*--- Extract the geometric sensitivities ---*/
 
+  const bool time_domain = config->GetTime_Domain();
+
   for (unsigned long iPoint = 0; iPoint < nPoint; iPoint++) {
 
-    su2double *Coord = geometry->nodes->GetCoord(iPoint);
+    auto Coord = geometry->nodes->GetCoord(iPoint);
 
     for (unsigned short iDim = 0; iDim < nDim; iDim++) {
 
-      su2double Sensitivity;
-
-      if(config->GetMultizone_Problem()) {
-        Sensitivity = geometry->nodes->GetAdjointSolution(iPoint, iDim);
-      }
-      else {
-        Sensitivity = SU2_TYPE::GetDerivative(Coord[iDim]);
-        /*--- Set the index manually to zero. ---*/
-        AD::ResetInput(Coord[iDim]);
-      }
+      su2double Sensitivity = geometry->nodes->GetAdjointSolution(iPoint, iDim);
+      AD::ResetInput(Coord[iDim]);
```

Comment:
For whatever reason this reset is fundamental for the correctness of multiple recordings with different inputs.
Reseting the indices (explicitly) during the CLEAR_INDICES recording does not work :mage:

Reply:
If @jblueh is at the DEV-meeting I will prob ask about this ResetInput and CLEAR_INDICES-recording (again 😞 🐔 🧠 ) to make sure I get how and why things work